src/users/dto/create-user.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsEmail, IsNotEmpty, IsString } from 'class-validator';
+
+export class CreateUserDto {
+  @IsEmail()
```

Comment:
**praise**: `@IsEmail()` is a great choice for validating the `email` field, ensuring it follows the correct format.